src/app/pages/CreateWalletPage/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -165,7 +165,10 @@ exports[`<CreateWalletPage  /> should match snapshot 1`] = `
   -webkit-flex-direction: column;
   -ms-flex-direction: column;
   flex-direction: column;
-  padding: 6px;
+  padding-left: 12px;
```

Comment:
This affect `This is your mnemonic` section not sure it is intended change.

Reply:
I noticed, but didn't care :shrug: